configuration/forms.md

```diff
@@ -565,3 +565,107 @@ If using a [XLSForm](http://xlsform.org/) and using [`medic-conf`](https://githu
 | `start` | A timestamp of when the form entry was started. |
 | `end` | A timestamp of when the form entry ended. |
 | `today` | Day of the survey. |
+
+
+# Multimedia in Forms
+
+## Multimedia Formats
+
+There are many supported formats for video, audio, and images. We recommend using h.264(mpeg) for video, jpeg for images, and mp3 for audio. When creating videos or images keep in mind the dimensions and storage capabilities on phones that may be used. Lower end phones have smaller storage and screen sizes. 
```

Comment:
I would mention something at the start of this paragraph about _how_ the media is rendered, eg: "When rendering images, video, and audio the CHT uses the browser's built in rendering tools. This means you can render any media format that is supported by the [minimum version of Chrome](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md#dependencies).".